frontend/src/stores/focusWell.js

```diff
@@ -28,6 +28,13 @@ export const useWellStore = defineStore('focusWell', {
         */
     }),
     getters: {
+        getIdProduct(state) {
+            if (state.well && !isNull(state.well.id_product)) {
+                return state.well.id_product
+            } else {
+                return null
+            }
+        },
         getRunAndLabel(state) {
```

Comment:
Is this function redundant?

Reply:
It's still used by `WellsByStatus.vue`, but now I look at it, I think that invocation is entirely redundant. Watch this space

Reply:
and changed.